### PR TITLE
remove methods that are no longer used when importing valuesets from …

### PR DESCRIPTION
**Pull request by dczulada**
…VSAC XML.

cqm-parsers does not call "load_from_xml" like bonnie-bundler had done.  These methods are no longer needed

Pull requests into cqm-models require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Code diff has been done and been reviewed
- [x] Tests have been run locally and pass

**Bonnie Reviewer:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Cypress Reviewer:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: lib/generate_types.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@ def self.generate_facility_location
 
     def self.generate_code_field
       # TODO: use code with all parameters, possibly randomize parameter values and optional information
-      QDM::Code.new('1234', 'SNOMED-CT')
+      QDM::Code.new('1234', 'SNOMEDCT')
```
Comment (okeefm) [inline lib/generate_types.rb]: The cqm-parsers `CODE_SYSTEMS` hash has the name of this code system as `SNOMED-CT`. Do we want this to be different than that?

Comment (dczulada) [reply lib/generate_types.rb]: Well...more important question...is "CodeSystemHelper" now dead code in cqm-parsers now that VSACValueSetLoader just uses the valueset name from VSAC.

The CodeSystemHelper is used in the method "retrieve_code_system_for_model" in data_criteria.rb (which is called by the "to_model" method).  I'm not versed enough in cqm-parsers to know if this is also just old/dead code.

Comment (okeefm) [reply lib/generate_types.rb]: That makes sense. 

Given that, does it make sense to remove `SNOMED-CT` everywhere it exists in this repo? It's also present in some of the fixtures (patient/value set), and in the RSpec tests.
